src/Discord.Net.WebSocket/DiscordSocketClient.cs

```diff
@@ -5,6 +5,7 @@
 using Discord.Net.Udp;
 using Discord.Net.WebSockets;
 using Discord.Rest;
+using Discord.WebSocket;
```

Comment:
This namespace shouldn't need to be imported given that the `DiscordSocketClient` class is already defined within it.